app/src/main/AndroidManifest.xml

```diff
@@ -9,8 +9,8 @@
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
-        android:theme="@style/AppTheme">
-        <activity android:name=".HelloWorld">
+        android:theme="@style/MainAppTheme">
+        <activity android:name=".MainActivity">
```

Comment:
aj v <application> elemente mozes deklarovat class pre Application pomocou bodky a nemusis vypisovat full cestu (nemohla som ti to napisat k inicialnemu commitu). to pred bodkou je vlastne package name